projects/plugins/jetpack/modules/likes.php

```diff
@@ -65,6 +71,8 @@ public static function init() {
 	 * Constructs Likes class
 	 */
 	public function __construct() {
+		$this->in_jetpack = class_exists( 'Jetpack' ) && method_exists( 'Jetpack', 'enable_module_configurable' );
```

Comment:
I would have loved if we had a utility function to do this consistently already, but seems like we don't. Plus, this is being done the same way in the publicize module, so I'm fine to do it the same way.